static/js/settings_org.js

```diff
@@ -868,6 +868,31 @@ function _set_up() {
     }
     realm_icon.build_realm_icon_widget(upload_realm_icon);
 
+    $('#deactivate_realm_button').on('click', function (e) {
+        if (!overlays.is_modal_open()) {
+            e.preventDefault();
+            e.stopPropagation();
+            overlays.open_modal('deactivate-realm-modal');
+        }
+    });
+
+    $('#do_deactivate_realm_button').on('click', function () {
+        if (overlays.is_modal_open()) {
+            overlays.close_active_modal();
+        }
```

Comment:
Does this conditional have any effect?

Reply:
Yes, I think in some cases it logs the error in console when we open a modal without closing existing modals (Which I think happened to while developing, that is why I've included this, I'll try to reproduce it again to check if it's still valid or not).

Reply:
Yeah, I sorta would suspect that this shouldn't be a thing, since this is the second step (inside the modal), not the thing that opens the modal, but worth checking.